release/long_running_tests/workloads/serve_failure.py

```diff
@@ -90,8 +102,11 @@ def __init__(self, max_deployments=1):
             (self.verify_deployment, 4),
         ]
         self.deployments = []
+        self.random_killer = RandomKiller.remote()
```

Comment:
nit: not really important here, but in general prefer the "dependency injection" pattern, which in this case means instantiate this actor outside and pass in the handle (it makes testing easier)